### PR TITLE
Add basic ffnet infrastructure

### PR DESCRIPTION
**Pull request by geigerzaehler**
We add a new Kubernetes cluster for the ffnet. This is the first step in tackling https://github.com/radicle-dev/radicle-registry/issues/290.

Follow-up

* Static IP addresses and DNS for boot nodes
* Add node instance that exposes RPC APIs
* Canary deployments of node of master branch builds

### CODE REVIEW

File: registry/ffnet/miners.tf [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          operator = "Exists"
+        }
+
+        container {
+          image   = local.node_image
+          name    = "radicle-registry-node"
+          command = ["radicle-registry-node"]
+          args = [
+            # The SS58 address for the seed string //Mine
+            "--mine=5HYpUCg4KKiwpih63PUHmGeNrK2XeTxKR83yNKbZeTsvSKNq",
```
[the remark this answers is not in the document]
Comment (geigerzaehler) [reply registry/ffnet/miners.tf:58]: Good point. Will create another address from a private key

File: registry/ffnet/main.tf [hunk 2]
```diff
@@ -0,0 +1,54 @@
+variable "project" {
+  type = string
+}
+
+locals {
+  node_image = "gcr.io/opensourcecoin/radicle-registry/node:4f5addddcec7b73a68f51eff9e88cdbba88b1780"
```
[the remark this answers is not in the document]
Comment (geigerzaehler) [reply registry/ffnet/main.tf:6]: Yes, we should make it automatic at some point. There is an issue (#79) for that.

File: registry/ffnet/miners.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (geigerzaehler) [reply registry/ffnet/miners.tf]: Done

[the remark this answers is not in the document]
Comment (CodeSandwich) [reply registry/ffnet/miners.tf]: Great! Is it made from a completely random key or do we have some access to it?

[the remark this answers is not in the document]
Comment (geigerzaehler) [reply registry/ffnet/miners.tf]: I’ve added a comment to the top of `miners.tf` file. You can find the secret key in `registry/secrets.yaml`.

[the remark this answers is not in the document]
Comment (CodeSandwich) [reply registry/ffnet/miners.tf]: I'm able to access this file when I'm not logged into GitHub. This means, that the miner account is as hijackable as it was, just a bit more obscure.

[the remark this answers is not in the document]
Comment (geigerzaehler) [reply registry/ffnet/miners.tf]: The `registry/secrets.yaml` file and its structure is visible but the _values_ are encrypted. We’re using [sops](https://github.com/mozilla/sops) for this.

**Pull request merged**